tokio/src/runtime/metrics/runtime.rs

```diff
@@ -47,6 +47,29 @@ impl RuntimeMetrics {
         self.handle.inner.num_workers()
     }
 
+    /// Returns the current number of active tasks in the runtime.
+    ///
+    /// This value increases and decreases over time as tasks are spawned and as they are completed or cancelled.
+    ///
+    /// To see the total number of spawned tasks, see `spawned_tasks_count`. Note that this API currently requires using `--cfg tokio_unstable`.
```

Comment:
If this is being stabilized, then the documentation should no longer say that it's unstable.